lib/sign_in/widgets/form.dart

```diff
@@ -56,21 +56,19 @@ class _SignInFormState extends State<SignInForm> {
           Padding(
             padding: EdgeInsets.symmetric(horizontal: 32),
             child: SvgPicture.asset(
-              services
-                  .config
-                  .assetName(context, 'logo/logo_with_text.svg'),
+              services.config.assetName(context, 'logo/logo_with_text.svg'),
               height: 64,
               alignment: Alignment.bottomCenter,
             ),
           ),
           SizedBox(height: 32),
           PrimaryButton(
-              onPressed: () => browser.open(
-                  url: services.config.webUrl('login'),
-                  options: InAppBrowserClassOptions()),
-              child: Text(s.signIn_form_signIn)),
+            onPressed: () => browser.open(
+                url: services.config.webUrl('login'),
+                options: InAppBrowserClassOptions()),
```

Comment:
What do you think about hiding the toolbar to not confuse the user?
```suggestion
            onPressed: () => browser.open(
              url: services.config.webUrl('login'),
              options: InAppBrowserClassOptions(
                inAppBrowserOptions: InAppBrowserOptions(toolbarTop: false),
              ),
            ),
```